addon/tailwind/components/base.css

```diff
@@ -21,6 +21,7 @@ making styling changes in the library */
 }
 
 *:focus {
+  outline:  var(--dodger-blue);
```

Comment:
I wouldn't say anything, but as long as you cleaned up some double-spaces elsewhere in the diff, could you get this one as well?